nav2_system_tests/src/updown/nav2_params.yaml

```diff
@@ -60,7 +60,7 @@ bt_navigator_rclcpp_node:
   ros__parameters:
     use_sim_time: True
 
-dwb_controller:
+nav2_controller:
```

Comment:
Theres are mostly dwb controller parameters, not controller parameters. There should probably be 2 set of params: those for the controller server (rate to execute, name of plugin to use, etc) and the controller (all the settings for DWB)

Reply:
I will do a follow up PR for this.